src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -3974,3 +3973,62 @@ instruct insertF(vReg dst, vReg src, vRegF val, immI idx, vReg tmp1, vReg tmp2,
   %}
   ins_pipe(pipe_slow);
 %}
+
+// ------------------------------ Vector shuffle -------------------------------
+instruct loadshuffleB(vReg dst, vReg src)
+%{
+  predicate(UseSVE > 0 && n->bottom_type()->is_vect()->element_basic_type() == T_BYTE);
+  match(Set dst (VectorLoadShuffle src));
+  ins_cost(SVE_COST);
+  format %{ "sve_mov $dst, $src\n\t" %}
+  ins_encode %{
+    __ sve_orr(as_FloatRegister($dst$$reg), as_FloatRegister($src$$reg), as_FloatRegister($src$$reg));
+  %}
```

Comment:
No need to emit the move if dst and src are the same register?

Reply:
OK. Thanks. I will fix that.